packages/sdk/src/cosmwasmclient.ts

```diff
@@ -96,23 +98,29 @@ export interface ExecuteResult {
 }
 
 export class CosmWasmClient {
-  public static makeReadOnly(url: string): CosmWasmClient {
-    return new CosmWasmClient(url);
+  public static makeReadOnly(url: string, feeTable?: Partial<FeeTable>): CosmWasmClient {
```

Comment:
Why does this constructor take the argument? A `CosmWasmClient` from here cannot sign, which probably means it does not need fees.

Reply:
Good point